auth0/src/test/java/com/auth0/android/provider/OAuthManagerTest.java

```diff
@@ -130,22 +130,4 @@ public void shouldHaveInvalidStateWhenOneIsNull() {
         exception.expect(AuthenticationException.class);
         OAuthManager.assertValidState("0987654321", null);
     }
-
-    @Test
-    public void shouldHaveValidNonce() {
-        OAuthManager.assertValidNonce("1234567890", "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJub25jZSI6IjEyMzQ1Njc4OTAifQ.oUb6xFIEPJQrFbel_Js4SaOwpFfM_kxHxI7xDOHgghk");
```

Comment:
This static method was removed. But the scenario is covered in `WebAuthProviderTest`